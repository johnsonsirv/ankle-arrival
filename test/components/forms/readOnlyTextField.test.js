/* eslint-disable react/react-in-jsx-scope */
/* eslint-disable no-undef */
import ReadOnlyTextField from '../../../src/components/forms/inputs/readOnlyTextField';

describe('<ReadOnlyTextField /> rendering', () => {
  const enzymeWrapper = mount(<ReadOnlyTextField />);
  it('renders correctly', () => {
    expect(enzymeWrapper).toMatchSnapshot();
  });
  it('should render 1 <input> text as readonly', () => {
    expect(enzymeWrapper.find('input').prop('readOnly')).toBe(true);
  });
});
