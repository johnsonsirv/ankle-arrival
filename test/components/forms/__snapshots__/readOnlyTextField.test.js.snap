// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ReadOnlyTextField /> rendering renders correctly 1`] = `ReactWrapper {}`;
